components/cost-saving-counter/spec/CostSavingCounter.ts

```diff
@@ -0,0 +1,15 @@
+import { createElement as h } from 'react';
+import { mount } from 'enzyme';
+import CostSavingCounter from '../src/CostSavingCounter';
+import toJson from 'enzyme-to-json';
+
+describe('CostSavingCounter', () => {
+  describe('when given valid props', () => {
+    const group = 'the public';
+    const component = mount(h(CostSavingCounter, { costsaving: 2345.56, group: group }));
+
+    it('renders cost saving', () => expect(component.find('h2').text()).toEqual('£2,345.56'));
+    it('renders the group', ()=> expect(component.find('p').text()).toContain(group));
+    it('contains no regressions', () => expect(toJson(component)).toMatchSnapshot());
```

Comment:
I don't think we should do snapshot testing like this.

I'd rather rely on Chromatic for visual regressions. (Of course there's an outstanding issue to fix that.)

Reply:
Storybook also offers built in support for this if we did want to do it. But I think it's worth reading the note at the bottom of the following page that sort of warns against it: https://storybook.js.org/docs/react/workflows/snapshot-testing

Reply:
Yes, Chromatic definitely has some good advantages! I know you were hitting the free snapshot limit - do you think a 2-tiered approach might help? First stage: use `storyshot` to suggest some attention maybe needed (before push)... second stage: visual regressions and PR. @daniel-ac-martin